src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -937,6 +937,23 @@ public override TypeInfo GetTypeInfo(SelectOrGroupClauseSyntax node, Cancellatio
             return GetTypeInfoForQuery(bound);
         }
 
+        /// <summary>
+        /// Gets the symbol information for the property of a sub-property pattern.
+        /// </summary>
+        public override SymbolInfo GetSymbolInfo(SubPropertyPatternSyntax node, CancellationToken cancellationToken)
+        {
+            var boundNode = GetLowerBoundNode(node) as BoundSubPropertyPattern;
+            if (boundNode != null)
+            {
+                var property = boundNode.Property;
+                return new SymbolInfo(property, boundNode.ResultKind == LookupResultKind.Viable ? CandidateReason.None : boundNode.ResultKind.ToCandidateReason());
+            }
+            else
+            {
+                return default(SymbolInfo);
```

Comment:
I'm aware of the existing inconsistency - there is that one place where we do it differently. I'm not intending to address that inconsistency now, nor do I want to add to it.